pkg/agent/evictionmanager/plugin/memory/numa_pressure.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/kubewharf/katalyst-core/pkg/client"
```

Comment:
why we must modify so much for eviction logic? we have currently made it online running for this logic, so if you want to modify this, please contact with @zzzzhhb and make sure you have fully tested (compared with current version) and access permission from @zzzzhhb

Reply:
These modifications do not involve functional changes, but only extract common public functions.

Reply:
you should confirm this with @zzzzhhb